Automattic/wp-calypso
# State: Remove unused recommended plugins state

#### Changes proposed in this Pull Request

Since #59191 we no longer use the recommended plugins state. This PR cleans it up.

#### Testing instructions

* Verify Calypso builds well.
* Verify that removed code is not in use.
* Verify all tests pass.
